### PR TITLE
Feature/category

### PR DESCRIPTION
**Pull request by annielin28815**
1. add category controller
2. add style of homepage
3. fix activity's button bug

### CODE REVIEW

**Review by samliuforwork — APPROVED** [review]
checked

File: app/views/qr_codes/attand_list.html.erb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="attand-list container text-center">
   <h3>活動名稱：<%= Activity.find(@attand_list_activity_id).title %></h3>
-  <span>目前報到人數：<%= @qr_codes.count %>/ <%= Ticket.where(event: @attand_list_activity_id).count %> </span>
+  <span>目前報到人數：
+  <%#= @qr_codes.count %>/ <%#= Ticket.where(event: @attand_list_activity_id).count %>
```
Comment (xx4673) [inline app/views/qr_codes/attand_list.html.erb:4]: 這裡為何註解掉？

**Review by xx4673 — APPROVED** [review]
checked

**Review by kihifung — APPROVED** [review]
checked

**Pull request merged**